pages/auth_zero.py

```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from pages.page import Page
+
+
+class AuthZero(Page):
```

Comment:
nit: I would prefer to follow the same naming for auth0. Please replace all references like ``AuthZero`` and ``auth_zero`` to ``auth0`` or ``Auth0``.